doc/sphinxman/source/introduction.rst

```diff
@@ -472,114 +472,85 @@ For more details, see Tables :ref:`Energy <table:energy_gen>`,
+    | CISD                    | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           | E          |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | QCISD                   | RHF            | CONV              | ---          | ---            | threaded [#f3]_             |           | E          |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | QCISD(T)                | RHF            | CONV              | ---          | ---            | threaded [#f3]_             |           | E          |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | CI(n)                   | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | FCI                     | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Mk-MRPT2                | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Mk-MRCCSD               | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Mk-MRCCSD(T)            | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | RAS-CI                  | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | CASSCF, RASSCF          | RHF/ROHF       | CONF/DF           | ---          | ---            | partially threaded          |           |            |
```

Comment:
Just CONF->CONV, but yes for two step we have both.

Cheers,
Daniel Smith



> On Apr 27, 2016, at 3:09 AM, Lori A. Burns <notifications@github.com> wrote:
> 
> In doc/sphinxman/source/introduction.rst <https://github.com/psi4/psi4/pull/370#discussion_r61219079>:
> 
> > +    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
> > +    | QCISD(T)                | RHF            | CONV              | ---          | ---            | threaded [#f3]_             |           | E          |
> > +    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
> > +    | CI(n)                   | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
> > +    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
> > +    | FCI                     | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
> > +    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
> > +    | Mk-MRPT2                | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
> > +    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
> > +    | Mk-MRCCSD               | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
> > +    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
> > +    | Mk-MRCCSD(T)            | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
> > +    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
> > +    | RAS-CI                  | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
> > +    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
> > +    | CASSCF, RASSCF          | RHF/ROHF       | CONF/DF           | ---          | ---            | partially threaded          |           |            |
> @dgasmith <https://github.com/dgasmith>, is CONF/DF right for casscf & rasscf? Any other changes amongst the detci-driven lines?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/psi4/psi4/pull/370/files/eac7691cb5802662e7ba6ec9275cca13aff28443..c0e5d325a349de1a4d77bda848678bc5eb794681#r61219079>